### DISCUSSION TITLE
How to limit how many past conversation are resent when asking a new prompt?

### DISCUSSION THREAD

**Discussion by evandrocoan** [danny-avila/LibreChat#2308] [Q&A] [answered]
I use a single conversation to ask chatgpt to explain many different sentences in a foreign language. I do so in the same prompt so I can quickly search If I already had asked it to explain some sentence before by searching for it. By looking at the logs, I see that librechat is resending all my previous messages and answers, which significantly increases the prompt tokens charged. Can I limit how many old/previous prompts are resent to each new prompt in a conversation?

**Comment by danny-avila** [comment, marked as answer]
Not yet but I'm working on this exact feature this week. Stay tuned

**Reply by evandrocoan** [reply]
Thanks. It would be nice if this setting were a preset setting instead of a global one so I could see just how much "memory" each chat would have. Perhaps something like:
![image](https://github.com/danny-avila/LibreChat/assets/5332158/6b17aed7-930d-4eaf-af94-b00524a14b72)
